test/e2e/test_tools.py

```diff
@@ -68,23 +67,23 @@ def test_get(self) -> None:
         group_descriptions_received = []
         group_selfjoin_received = []
         group_status_received = []
-        for group in groups_expected:
+        for group in sorted(groups_expected, key=lambda x: x["name"]):
```

Comment:
I added some groups with more diverse characteristics. It is now necessary to sort the groups by name